common/lib/xmodule/xmodule/fields.py

```diff
@@ -252,3 +253,19 @@ def enforce_type(self, value):
             return value
 
         return self.from_json(value)
+
+
+class ScoreField(JSONField):
+    def from_json(self, value):
+        if value is None:
+            return value
+        if isinstance(value, Score):
+            return value
```

Comment:
https://build.testeng.edx.org/job/edx-platform-python-unittests-pr/38303/ the two failures.